PHP fails to build on Debian

## Issue description

```
...
checking for LDAP support... yes
checking for LDAP Cyrus SASL support... /nix/store/5mfw60742kf18wlsd6f72d3pzm6bjxcr-cyrus-sasl-2.1.26-dev
checking size of long int... 8
configure: error: Cannot find ldap libraries in /usr/lib.
builder for ‘/nix/store/jsgb3gdcqhmmcav7qfc5k0kxn7703nrv-php-5.6.27.drv’ failed with exit code 1
...
```

### Steps to reproduce
Build PHP on non-NixOS. Both PHP 5.6 and PHP 7 fail to build for the same reason.


## Technical details

* System: Debian
* Nixpkgs version: 16.09

--- a/pkgs/development/interpreters/php/default.nix
+++ b/pkgs/development/interpreters/php/default.nix
@@ -50,7 +50,7 @@ let
 
         ldap = {
           configureFlags = [
-            "--with-ldap"
+            "--with-ldap=/invalid/path"
             "LDAP_DIR=${openldap.dev}"
             "LDAP_INCDIR=${openldap.dev}/include"
             "LDAP_LIBDIR=${openldap.out}/lib"

